viewing_party/party.py

```diff
@@ -1,23 +1,171 @@
 # ------------- WAVE 1 --------------------
+"""Create and modify movie data"""
+
 
 def create_movie(title, genre, rating):
-    pass
+    """create a movie dictionary"""
+    if title is None or genre is None or rating is None:
+        return None
+    return {"title": title, "genre": genre, "rating": rating}
+
+
+def add_to_watched(user_data, movie):
+    """append movie dictionary to user_data "watched" list"""
+    user_data["watched"].append(
+        create_movie(movie["title"], movie["genre"], movie["rating"])
+    )
+    return user_data
+
+
+def add_to_watchlist(user_data, movie):
+    """append movie dictionairy to user_data "watchlist" list"""
+    user_data["watchlist"].append(

```

Comment:
[suggestion] When working with dictionary keys (which can be typo prone) it's sometimes helpful to make constants that our editor is better able to suggest for us. If we had

```python
KEY_WATCHLIST = "watchlist"
```

defined elsewhere, we could write

```python
    user_data[KEY_WATCHLIST].append(
```

We might get better suggestions from our editor, reducing typos. If in the future a key ever needed to change, we'd also only need to change it in one place. Refactoring shortcuts can also help, in a good editor, but sometimes we can help things along ourselves!